ludwig/schema/features/preprocessing/bag.py

```diff
@@ -54,10 +54,14 @@ class BagPreprocessingConfig(BasePreprocessingConfig):
         parameter_metadata=FEATURE_METADATA[BAG][PREPROCESSING]["lowercase"],
     )
 
-    most_common: int = schema_utils.PositiveInteger(
+    most_common: int = schema_utils.OneOfOptionsField(
```

Comment:
Make sure to make the type annotation `Union[int, float]`

But if the value can be an integer or a float, I think it would be simpler to just let the type be a float and if the user enters a whole number to just treat it as an integer. WDYT?

Reply:
Good catch! I think `Union[int, float]` is clearer than just having `float` in this case.